Examples/Swift/ViewController.swift

```diff
@@ -352,8 +352,27 @@ extension ViewController: NavigationViewControllerDelegate {
     }
 }
 
+/**
+ To find more pieces of the UI to customize, checkout DayStyle.swift and NightStyle.swift
```

Comment:
Nit: Both DayStyle and NightStyle resides in `DayStyle.swift`.  Extract the night style or edit the comment.